docs/src/learning_rate_scheduler.md

```diff
@@ -13,23 +13,25 @@ We shall compare the following configurations of implemented schedulers.
 4. Misfit controlling timestep (Terminating) `DataMisfitController()` [Iglesias & Yang 2021](https://doi.org/10.1088/1361-6420/abd29b) - purple
 5. Misfit controlling timestep (Continuing beyond Terminate condition) `DataMisfitController(on_terminate="continue")` - brown
 
-One can define the schedulers as
-```julia
-scheduler = DefaultScheduler(0.5) # fixed stepsize, default values: 1
-```
-Then when constructing an EnsembleKalmanProcess, one uses the keyword argument
+!!! info "Recommended Scheduler"
+    For typical problems [we provide a default scheduler](@ref defaults) depending on the process. For example, when constructing an `Inversion()`-type `EnsembleKalmanProcess`, by default this effectively adds the scheduler
+    ```julia
+    scheduler = DataMisfitController(terminate_at = 1) # adaptive step-sizestop at algorithm time "T=1"
+    ```
+
+To modify the scheduler, use the keyword argument
 ```julia
 ekpobj = EKP.EnsembleKalmanProcess(args...; scheduler = scheduler, kwargs...)
 ```
-A variety of other schedulers can be defined similarly:
+Several choices are available:
 ```julia
 scheduler = MutableScheduler(2) # modifiable stepsize
 scheduler = EKSStableScheduler(numerator=10.0, nugget = 0.01) # Stable for EKS
-scheduler = DataMisfitController(on_terminate = "continue") # non-terminating
+scheduler = DataMisfitController(terminate_at = 1000) # stop at algorithm time "T=1"
```

Comment:
Assuming you meant to write `terminate_at = 1`?

Reply:
Done